lib/mjsonwp.js

```diff
@@ -1,12 +1,14 @@
 import _ from 'lodash';
 import { validators } from './validators';
-import { NotYetImplementedError, BadParametersError } from './errors';
+import { NotYetImplementedError, NotImplementedError, BadParametersError } from './errors';
 import { METHOD_MAP, NO_SESSION_ID_COMMANDS } from './routes';
 
 class MJSONWP {}
-
 function buildValidatedCommand (command) {
   return async function (...args) {
+    if(!command) {
```

Comment:
i don't think we should do this check in this function, but rather below when we are unwrapping the command map. so here:

```js
try {
  if (!spec.command) throw new NotImplementedError();
  checkParams(spec.payloadParams, jsonObj);
```